bots/machine/testvm.py

```diff
@@ -177,7 +180,13 @@ def __init__(self, address="127.0.0.1", image=None, verbose=False, label=None, b
         else:
             self.web_address = browser
             self.web_port = 9090
-        self.label = label or self.image + "-" + self.ssh_address + "-" + self.ssh_port
+        if label:
+            self.label = label
+        elif self.image is not "unknown":
+            self.label = self.image + "-" + self.ssh_address + "-" + self.ssh_port
+        else:
+            self.label = "DIRECT"
```

Comment:
How about "user@address:port"? This might be more useful when connecting to multiple machines.